### ISSUE TITLE
Exhibit page heading elements are not in a sequentially-descending order

### ISSUE THREAD

**Issue by thanyabegum** [pulibrary/dpul#1220]
Content blocks in the Capturing Feathers exhibit pages are composed of ```<h3>``` tags when they should probably be ```<p>``` tags to be more semantically accurate. This issue may exist in other exhibits.

<img width="476" alt="Screen Shot 2021-11-05 at 3 27 00 PM" src="https://user-images.githubusercontent.com/66143640/140567519-ac613c89-a7a2-4dce-bc52-07010ee48a01.png">

[Learn more about this failing Lighthouse test.](https://web.dev/heading-order/?utm_source=lighthouse&utm_medium=devtools)

**Comment by hackartisan** [comment]
We should talk to @kelea99 about this. This content is managed through the front-end UI by curators. Maybe we could try changing the h3s to paragraphs in one exhibit, and see how it looks. Consider whether there is a training element to this.
